### DISCUSSION TITLE
[mousetrap] Bad typing for bind callback

### DISCUSSION THREAD

**Discussion by brunnre8** [DefinitelyTyped/DefinitelyTyped#63233] [Issues with a @types package]
[[mousetrap] Return type of callback should be `void…](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/84c35a31940048dc308fea3c1501856aa77fb11b) is a bogus commit and needs to be reverted.

The callback is not always void, False can be returned to prevent the default handler:

https://craig.is/killing/mice#:~:text=As%20a%20convenience%20you%20can%20also%20return%20false%20in%20your%20callback

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
fixed in #63238
